external/tomcat/test.properties

```diff
@@ -4,7 +4,4 @@ home_path=""
 tag_version="10.0.7"
 ant_version="1.10.6"
 openssl_version="1.1.1d"
-debian_packages="git wget tar make gcc linux-libc-dev libc6-dev perl"
-alpine_packages="bash git wget tar make gcc libc-dev perl linux-headers"
-centos_packages="git wget tar make gcc glibc-devel perl"
-ubi_packages="git wget tar make gcc glibc-devel"
+ubuntu_packages="git wget tar make gcc glibc-devel"
```

Comment:
Same as following question any reason change the `git wget tar make gcc linux-libc-dev libc6-dev perl` to `git wget tar make gcc glibc-devel`?